cpp/benchmarks/io/orc/orc_reader_benchmark.cpp

```diff
@@ -40,7 +40,7 @@ void BM_orc_read_varying_input(benchmark::State& state)
   cudf::size_type const run_length  = state.range(2);
   cudf_io::compression_type const compression =
     state.range(3) ? cudf_io::compression_type::SNAPPY : cudf_io::compression_type::NONE;
-  auto const source_type = static_cast<io_type>(state.range(4));
+  io_type const source_type = static_cast<io_type>(state.range(4));
```

Comment:
This commit seems to have reverted no trailing return type, but use of `auto` on static_cast assignment, as well as reverting `[[nodiscard]]`.

Reply:
Sorry, I had to manually fix the `git revert` on a couple commits that had conflicts on like 200 files. There is only an auto-fix, not an auto-undo-fix. I will rerun the script and apply the changes.

Reply:
Fixed